core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestTransformCorrelatedScalarSubquery.java

```diff
@@ -186,6 +189,8 @@ public void rewritesScalarSubquery()
                 .on(p -> p.correlatedJoin(
                         ImmutableList.of(p.symbol("corr")),
                         p.values(p.symbol("corr")),
+                        INNER,
+                        TRUE_LITERAL,
```

Comment:
Why this change?

if i revert it, the `correlatedJoin` method being called supplements INNER, TRUE_LITERAL by default
so it looks redundant to me